lib/api/CMakeLists.txt

```diff
@@ -101,6 +101,7 @@ if(WASMEDGE_BUILD_STATIC_LIB)
   wasmedge_add_static_lib_component_command(wasmedgeValidator)
   wasmedge_add_static_lib_component_command(wasmedgeExecutor)
   wasmedge_add_static_lib_component_command(wasmedgeHostModuleWasi)
+  # wasmedge_add_static_lib_component_command(wasmedgeHostModuleWasiNN)
```

Comment:
Why comments this out?

Reply:
I think the static library not need this now. We should refactor the whole static library architecture in the future.